CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.1.0
+
+- Adds Subscriber Attributes, which allow developers to store additional, structured information
+for a user in RevenueCat. More info: https://docs.revenuecat.com/docs/user-attributes.
+
```

Comment:
Add reference to invalidate Purchaser info cache

Reply:
maybe we can start doing the link to release changelogs thing?